.github/workflows/quick.yaml

```diff
@@ -159,17 +169,19 @@ jobs:
 
   CodeQL-tests:
 
-    runs-on: [ ubuntu-22.04 ]
+    runs-on: [ ubuntu-24.04 ]
 
     permissions:
       security-events: write
 
     steps:
 
-      - name: Install Squid prerequisite Linux packages
+      - name: Install prerequisite Linux packages
```

Comment:
This minor out-of-scope change reduces step naming inconsistency that complicated this PR work. After this change, all package installing steps that require editing apt configuration get the same name.